Automattic/wp-calypso
# Jetpack New Pricing Page - For 820-1024px screens, render a special image for the "featured" Complete card

On 820-1024px screens, render a special image for the "featured" Complete card. 

#### Proposed Changes

* Jetpack Complete featured Card Hero Image is overflowing when the screen size is >821px. This PR uses a new special image on screens 820px-1024px only for Jetpack Complete. 
* Add a new breakpoint **md** in **HeroImage** component, map same "lg" image to "md" for all products, for Jetpack Complete map a new image for "md" breakpoint
* Use CSS to hide other screen size divs and show only the div of matching size

#### Testing Instructions


* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout fix/jetpack-complete-md`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1
* Switch to the Bundles section and resize the screen to anywhere between `820px - 1024px`
* Make sure a different image is displayed for Jetpack Complete (refer the screenshot below)
  
#### Screenshot
##### Before
![Screenshot 2022-09-22 at 7 43 36 PM](https://user-images.githubusercontent.com/2027003/191770811-1f29e4d9-334f-49a3-801f-70d3964f43ef.png)



##### After
![Screenshot 2022-09-22 at 7 42 23 PM](https://user-images.githubusercontent.com/2027003/191770832-f1e161de-0120-426c-970c-df47c68028b3.png)


#### Pre-merge Checklist






Related to #


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203005354786559
